src/python/tests/conftest.py

```diff
@@ -0,0 +1,68 @@
+# this defines some basic utilities shared among all of the tests
+#
+# DO NOT import pygrackle into global scope in this file (or in any file
+# imported by this file). Some tests need to be runable without installing
+# pygrackle
+
+import os
+from typing import NamedTuple
+
+import pytest
+
+
+# this hook is used to add more command line flags to the pytest launcher
+def pytest_addoption(parser):
+    parser.addoption(
+        "--answer-skip",
```

Comment:
I would prefer to only have `--answer-store` since they do the same thing. Both variables are mentioned above for running tests without answer testing. I think it just adds confusion.

Reply:
Sure I can do that!

I'm just spitballing here, but I think it could still be nice (but definitely not essential) to support disabling all of the answer-tests. If we aren't worried about backwards compatability, what if we made ``-local-dir`` a required argument to run the answer tests?[^1] In that picture:
- `pytest grackle/src/python` wouldn't run all non-answer tests
- `pytest grackle/src/python --answer-store --local-dir=path/to/answers` would store answers (and run all non-answer tests)
- `pytest grackle/src/python --answer-store` would abort with an error.
- `pytest grackle/src/python --local-dir=path/to/answers` would check answer-test answers (and run all non-answer tests)

To be clear, I'm definitely not wedded to this idea. I'm happy to move forward and just simply remove ``--answer-skip``. In fact, I'll just plan to go ahead and simply remove ``--answer-store`` without any sort of replacement unless I hear that you like this alternative idea.

[^1]: If we went that route, maybe we should rename `--local-dir` to `--answer-dir`

Reply:
Yeah, I like this. I also really like changing `--local-dir` to `--answer-dir`. "local-dir" can be interpreted in a variety of different ways, but to me "answer-dir" can only mean "the answers are/go here." If we do this, the meaning of "will not run answer tests" needs clarifying to either mean 1) the tests won't run at all or 2) the code to generate the answers will be run, but no comparison will happen.